charts/pega/templates/_helpers.tpl

```diff
@@ -122,6 +122,12 @@ until cqlsh -u {{ $cassandraUser | quote }} -p {{ $cassandraPassword | quote }}
 # Additional JVM arguments
 - name: JAVA_OPTS
   value: "{{ .node.javaOpts }}"
+- name: ADDITIONAL_CATALINA_OPTS
+{{- if .node.additionalCatalinaOpts }}
+  value: "{{ .node.additionalCatalinaOpts }}"
+{{- else }}
+  value: "-XX:MetaspaceSize=512M -XX:MaxMetaspaceSize=1G -XX:ReservedCodeCacheSize=512M -XX:+DisableExplicitGC -XX:+ParallelRefProcEnabled -XX:+UseStringDeduplication -Djava.awt.headless=true -Xlog:gc*,gc+heap=debug,gc+humongous=debug:file=/usr/local/tomcat/logs/gc.log:tags,uptime,pid,level,time"
```

Comment:
Would it make more sense to have the default values baked into the pega-web-ready docker image to minimize the risk of dropping these recommended params?  Also, the hardcoded path for gc logs in these params makes me think the default params/values should be delivered elsewhere.

Reply:
Instead of defaulting these values in templates, these need to be passed by users by setting them in values.yaml because this is a web-ready image which acts as base image for fullybaked images across all releases.

We can provide the ability to pass the "additionalCatalinaOpts" and append it to CATALINA_OPTS in docker-pega-web-ready (https://github.com/pegasystems/docker-pega-web-ready/blob/master/tomcat-bin/setenv.sh)